docs/site/Using-components.md

```diff
@@ -7,24 +7,43 @@ sidebar: lb4_sidebar
 permalink: /doc/en/lb4/Using-components.html
 summary:
 ---
-Components are an important part of LoopBack 4. We are keeping the core small and easy to extend, making it easy for independent developers to contribute additional features to LoopBack. Components are the vehicle for bringing more goodness to your applications.
 
-A typical LoopBack component is an [npm](https://www.npmjs.com/) package exporting a Component class which can be added to your application.
+Components play an important part in the extensibility of LoopBack 4.
+A Component makes it easy for independent developers to contribute additional
+features to LoopBack. Components serve as a vehicle to group extension
+contributions such as [Context Bindings](Context.md) and various Artifacts to allow easier
+extensibility of your Application.
 
-```js
-import {Application} from '@loopback/core';
-import {AuthenticationComponent} from '@loopback/authentication';
+A typical LoopBack component is an [npm](https://www.npmjs.com) package
+exporting a Component class which can be added to your application.
 
-const app = new Application();
+```ts
+import { RestApplication } from "@loopback/rest";
```

Comment:
The formatting does not conform to our style. We don't use extra spaces for the `{}`.